Cargo.toml

```diff
@@ -401,7 +402,7 @@ tree-sitter-jsdoc = { git = "https://github.com/tree-sitter/tree-sitter-jsdoc",
 tree-sitter-json = "0.20.2"
 tree-sitter-markdown = { git = "https://github.com/MDeiml/tree-sitter-markdown", rev = "330ecab87a3e3a7211ac69bbadc19eabecdb1cca" }
 tree-sitter-proto = { git = "https://github.com/rewinfrey/tree-sitter-proto", rev = "36d54f288aee112f13a67b550ad32634d0c2cb52" }
-tree-sitter-python = "0.20.2"
+tree-sitter-python = "0.21.0"
```

Comment:
Do we still need this grammar as a dependency?

Reply:
you are right, i should remove that dependency